packages/cli/package.json

```diff
@@ -63,6 +63,7 @@
     "shell-quote": "1.6.1",
     "strip-ansi": "^5.2.0",
     "sudo-prompt": "^9.0.0",
+    "unzipper": "^0.10.10",
```

Comment:
Can we unzip using Node's `zlib` https://nodejs.org/api/zlib.html? `unzipper` brings extra 1.4MB to node_modules, I'd like to avoid that

Reply:
I'll give it a try but I do worry about the size of some of those zips and making things correctly.

Reply:
Maybe there's a lighter alternative?

Reply:
How did you get the computed size? npm only reports 51.6kB which I imagine does not take into account dependencies:
![image](https://user-images.githubusercontent.com/606594/78073385-abcd6300-7355-11ea-8e08-c31b8f047030.png)

I have to support node 8, right? It looks like with node 10 I could use [`stream.pipeline`](https://nodejs.org/api/stream.html#stream_stream_pipeline_source_transforms_destination_callback) and have a clean implementation but I remember seeing a comment saying that node 8 will get drop at the same time as RN.

Reply:
It has some heavy deps. Check this site: https://packagephobia.now.sh/result?p=unzipper

Reply:
Node 8 will be dropped in the next RN version, since Metro 0.59 dropped it

Reply:
I've replaced it with [`node-stream-zip`](https://packagephobia.now.sh/result?p=node-stream-zip) that does not have any [external dependency](https://packagephobia.now.sh/result?p=node-stream-zip).

Thanks for the site!